pages/review.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Reviews</title>
+    <link rel="stylesheet" href="../css/style.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="container container--header">
+        <a class="logo" href="../index.html">
+          <span class="logo__accent">Code</span>Studio
+        </a>
+        <nav class="nav">
+          <ul class="nav__links">
+            <li><a class="nav__link" href="../index.html">Home</a></li>
+            <li><a class="nav__link" href="./pages/about.html">About Us</a></li>
+            <li><a class="nav__link" href="./pages/course.html">Our Course</a></li>
+            <li><a class="nav__link" href="#">Reviews</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="section reviewIntro">
+        <div class="container hero-container">
+          <h1 class="reviewTitle">Still in doubt?</h1>
+          <h2 class="reviewTextIntro">
+            What about hearing a little  bit more from our students themselves?
+            Check out the reviews below to see what they have to say!
+          </h2>
+        </div>
+      </section>
+      <section>
+        <div id="container reviews">
+          <div class="reviewCard">
+            <img 
+              src="../images/ProfileImg/bruna.jpg" 
+              alt="Woman with a black dress and face witth a lot os piercings" 
+              class="profileImg"
+            >
+            <div class="reviewText">
+              <h4>Bruna:</h4> 
+              <p>
+                <br> 
```

Comment:
You shouldn't use "br" for getting spaces between paragraphs. Instead you should use "margin" or "padding"